tests/testdata/test_addons/package_global_config_everything/package/appserver/static/js/build/custom/custom_input_row.js

```diff
@@ -0,0 +1,34 @@
+class CustomInputRow {
+  /**
+   * Custom Row Cell
+   * @constructor
+   * @param {Object} globalConfig - Global configuration.
+   * @param {string} serviceName - Input service name.
+   * @param {element} el - The element of the custom cell.
+   * @param {Object} row - custom row object,
+   *     use this.row.<field_name>, where <field_name> is a field name
+   */
+  constructor(globalConfig, serviceName, el, row) {
+    this.globalConfig = globalConfig;
+    this.serviceName = serviceName;
+    this.el = el;
+    this.row = row;
+  }
+
+  getDLRows() {
+    return Object.fromEntries(
+      Object.entries(this.row).map(([key, value]) => [
+        key,
+        key === "interval" ? `${value} sec` : value,
+      ])
+    );
+  }
+
+  render() {
```

Comment:
please add comment here that this function would be ignored and can be safely omitted due to presence of `getDLRows`

Reply:
We already mentioned in the docs that `render` has low priority, do we need to add the same in the code below the `note` section?

<img width="874" alt="Screenshot 2024-06-24 at 11 53 09 AM" src="https://github.com/splunk/addonfactory-ucc-generator/assets/163995910/0a0fa880-9bee-408a-a5a7-889ef608b86c">